evadb/executor/create_function_executor.py

```diff
@@ -219,24 +220,13 @@ def handle_forecasting_function(self):
             pickle.dump(model, f)
             f.close()
 
-        arg_map_here = {"model_name": model_name, "model_path": model_path}
-        function = self._try_initializing_function(impl_path, arg_map_here)
-        io_list = self._resolve_function_io(function)
+        io_list = self._resolve_function_io(None)
 
         metadata_here = [
+            FunctionMetadataCatalogEntry("model_name", model_name),
+            FunctionMetadataCatalogEntry("model_path", model_path),
             FunctionMetadataCatalogEntry(
-                key="model_name",
-                value=model_name,
-                function_id=None,
-                function_name=None,
-                row_id=None,
-            ),
-            FunctionMetadataCatalogEntry(
-                key="model_path",
-                value=model_path,
-                function_id=None,
-                function_name=None,
-                row_id=None,
+                "output_column_rename", self.node.outputs[0].name
```

Comment:
May I know the difference between this and `"output_column_rename", arg_map["predict"]`? Thanks!

Reply:
That is a good point. We can use `arg_map["predict"]`.